aruco_robot:
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  tcp_controller:
      type: effort_controllers/JointPositionController
      joint: joint_base_tcp
      pid: {p: 100.0, i: 0.01, d: 10.0}
